controllers/logout.go

```diff
@@ -0,0 +1,26 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	html "github.com/gutakk/go-google-scraper/helpers/html"
+	session "github.com/gutakk/go-google-scraper/helpers/session"
+)
+
+const (
+	logoutSuccessFlash = "You've been logged out"
```

Comment:
Always prefer non-contracted English spelling:

```suggestion
	logoutSuccessFlash = "You have been logged out"
```

The contracted spelling for spoken formats only.

Reply:
Fixed in e599ca2117f078213bcd7b9ac559b116d9fb3278